docker/images/install-dsa-deps.sh

```diff
@@ -12,6 +12,7 @@ sudo make install
 cd ../
 rm -rf idxd-config
 
+
 # Install DML Library
 git clone --recursive https://github.com/intel/DML.git
```

Comment:
Maybe instead of a dummy change we should actually specify a commit id / tag for the DML here? Since purpose of this change is to get newer version, it would be more clear

Reply:
updated to use https://github.com/intel/DML/commit/e44443c24d53552b248b9869b1b16f89cd970f52 